app/components/windows/EditStreamInfo.vue

```diff
@@ -25,6 +25,41 @@
         :loading="searchingGames"
         @search-change="debouncedGameSearch"
         @input="onGameInput"/>
+        <div  v-if="areAvailableProfiles">
```

Comment:
This is indented too far and there is an extra space after `<div`

Reply:
Done.